src/include/simeng/Elf.hh

```diff
@@ -29,10 +29,16 @@ class Elf {
   uint64_t getProcessImageSize() const;
   bool isValid() const;
   uint64_t getEntryPoint() const;
+  uint64_t getPhdrTableAddress() const;
+  uint64_t getPHENT() const;
+  uint64_t getPHNUM() const;
 
  private:
   uint64_t entryPoint_;
-  std::vector<ElfHeader> headers_;
+  std::vector<Elf64_Phdr> pheaders_;
+  uint16_t e_phentsize;
+  uint16_t e_phnum;
+  uint64_t phdrTableAddress_ = 0;
```

Comment:
Although not really concerning this PR, could comments be added to member variables and functions inline with other header files in the project. It is good to update/fix these things as we find them